woocommerce/woocommerce
# Standardize usage of NVM

### All Submissions:


### Changes proposed in this Pull Request:

As we converge on the creation of a monorepo it becomes increasingly necessary that we are all using the same version of Node and NPM when working on WooCommerce Core. When working on various PRs I've encountered issues caused by `package-lock.json` files in different packages having been generated with different versions of NPM. It is for this reason that I think we should adopt the use of [NVM](https://github.com/nvm-sh/nvm) in order to standardize our Node versions.

This PR adds a `.nvmrc` file which can be consumed by the `nvm use` and `nvm install` commands. ~~I've also adjusted Travis so that it will run the CI install command~~(See commits for explanation) as well as install the version of Node described in the `.nvmrc` file.

### How to test the changes in this Pull Request:

1. [Install NVM (if you don't already have it)](https://github.com/nvm-sh/nvm)
2. `nvm install` in the root WC directory
3. `rm -rf node_modules tests/e2e/env/node_modules tests/e2e/factories/node_modules tests/e2e/utils/node_modules`
4. `npm install` should quickly install dependencies for WooCommerce Core and then all of the packages.

### Other information:
